src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -14,12 +14,74 @@
 
 package com.google.sps.servlets;
 
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.api.services.youtube.YouTube;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 /**
- * Servlet that fetch from Youtube Server.
+ * Servlet that fetches from Youtube Server.
  */
-@WebServlet("/userLogIn")
+@WebServlet("/YouTubeComments")
 public class YoutubeServlet extends HttpServlet {
+  private static final Long COMMENT_LIMIT = 100L;
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  private static final String URL_PARAMETER = "url";
+  private static final String SNIPPET_PARAMETER = "snippet,replies";
```

Comment:
It is unclear to me what this variable has. Can you add a comment here to clarify?
I would also pluralize PARAMETER as it seems like you have multiple.

Reply:
Which variable are you referring to? And what do you mean by pluralize?

Reply:
SNIPPET_PARAMETER. Call it SNIPPET_PARAMETERS. I made the word pluralize up I guess...